src/components/account/accountComponent.test.js

```diff
@@ -1,55 +1,59 @@
 import React from 'react';
 import chai, { expect } from 'chai';
-import { spy } from 'sinon';
+import sinon, { spy, mock } from 'sinon';
 import sinonChai from 'sinon-chai';
 import { shallow, mount } from 'enzyme';
 import { Provider } from 'react-redux';
+import * as accountApi from '../../utils/api/account';
```

Comment:
You defined 'accountApi' and 'sinon' and 'mock' but you didn't use them. please remove them.